src/javascript/binary/static/common/common.utility.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Created by qingwei on 10/13/15.
```

Comment:
OK, btw, it's added by the IDE by default

On Thu, Oct 22, 2015 at 5:57 PM, Jean-Yves Sireau <notifications@github.com>
wrote:

> In src/javascript/binary/static/common/common.utility.js
> <https://github.com/binary-com/binary-static/pull/574#discussion_r42729837>
> :
>
> > @@ -0,0 +1,26 @@
> > +/**
> > + * Created by qingwei on 10/13/15.
>
> no need for "Created by qingwei on 10/13/15" comment; we can find out who
> wrote what with the 'git blame' function.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/binary-com/binary-static/pull/574/files#r42729837>.
>



-- 

Regards,
Qing Wei